File: php-poo/aulas/aula10/Funcionario.php
<?php
require_once 'Pessoa.php';
class Funcionario extends Pessoa
{
    // Atributes
    private $setor;
    private $trabalhando;

    // Method
    public function mudarTrabalho() {
        // TODO
        $this->trabalhando = ! $this->trabalhando;
    }

    // Special Mehods - Getters and Setters
    public function getSetor()
    {
        return $this->setor;
    }

    public function setSetor($setor): void
    {
        $this->setor = $setor;
    }

    public function getTrabalhando()
    {
        return $this->trabalhando;
    }

    public function setTrabalhando($trabalhando): void
    {
        $this->trabalhando = $trabalhando;
    }

}

File: java-poo/projects/poo-aula10/src/aula10/Funcionario.java
package aula10;

public class Funcionario extends Pessoa {
    // Attributes
    private String setor;
    private boolean trabalhando;

    // Methods
    public void mudarTrabalho() {
        this.trabalhando = !this.trabalhando;
    }

    // Special Methods - Getters and Setters
    public String getSetor() {
        return setor;
    }

    public void setSetor(String setor) {
        this.setor = setor;
    }

    // type boolean ->  Generate (Default) -> isTrabalhando()
    public boolean getTrabalhando() {
        return trabalhando;
    }

    public void setTrabalhando(boolean trabalhando) {
        this.trabalhando = trabalhando;
    }
}
